tests/test_cbv.py

```diff
@@ -70,27 +120,21 @@ class CBV:
             def g(self) -> bool:
                 return hasattr(self, "class_var")
 
-        client = TestClient(router)
+        return
+
+    def test_class_var(self, cbv_router: None, client: TestClient) -> None:
         response = client.get("/")
-        assert response.status_code == 200
+        assert response.status_code == SUCCESS_CODE
         assert response.content == b"false"

```

Comment:
```suggestion
        assert response.json() == False

```

Reply:
does not pass b"false" object is the object returned while decoding it makes in a "false" string- there is no way to convert it to true boolean